crates/ef-testing-proc-macro/src/constants.rs

```diff
@@ -0,0 +1,9 @@
+use std::path::{Path, PathBuf};
+
+lazy_static::lazy_static! {
+    pub(crate) static ref TEST_FILTER_PATH: PathBuf =  Path::new("blockchain-tests-skip.yml").to_path_buf();
+    pub(crate) static ref REGEX: regex::Regex = regex::Regex::new(r"[=^.]").unwrap();
```

Comment:
why do we need this? 
the name isn't so descriptive, maybe we should rename it

Reply:
true, the regexp is used to filter out characters that appear in the ef-tests but are not valid for rust identifiers.